oak-store-spi/src/main/java/org/apache/jackrabbit/oak/json/JsonSerializer.java

```diff
@@ -300,11 +300,11 @@ private static class JsonFilter {
 
         private static final Pattern EVERYTHING = Pattern.compile(".*");
 
-        private final List<Pattern> nodeIncludes = newArrayList(EVERYTHING);
+        private final List<Pattern> nodeIncludes = new ArrayList<>(Arrays.asList(EVERYTHING));
```

Comment:
Maybe Collections.singleton() instead of Arrays.asList()  is more straightforward. Or List.of() ?

Reply:
This was tested, and I believe it proved to require modification.

Reply:
It's still an ArrayList.

Reply:
and it can't just be `Arrays.asList(EVERYTHING)` ?